### PR TITLE
Typography,avatar,badge

### PR DESCRIPTION
**Pull request by xsaurabhsuthar**
Added Avatar, Badge and Typography component.

### CODE REVIEW

**Review by muskaanshah — APPROVED** [review]
Code looks good, maybe you can explicitly mention things in the documentation site rather just just the examples and code so that user understands better

**Comment by xsaurabhsuthar** [comment]
Yes, I need to do that because there are so many options in every component. Thanks for the review.

File: docs.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  grid-template-areas: "header header" "aside main";
+  grid-template-rows: 5rem 1fr;
+  grid-template-columns: 15%;
+  box-sizing: border-box;
+}
+
+main {
```
Comment (ScoobyDubeyDo) [inline docs.css]: Avoid grabbing elements in a component library, try using classes in place of it. Doing so can affect the user's styling.

File: docs.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  grid-area: main;
+  overflow-y: scroll;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 30px 50px;
+}
+
+aside {
```
Comment (ScoobyDubeyDo) [inline docs.css]: Again don't grab elements.

**Review by ScoobyDubeyDo — CHANGES_REQUESTED** [review]

File: docs.css (hunk 2 above)
Comment (xsaurabhsuthar) [reply docs.css]: Is it okay if I'm using only in docs page? Because I haven't used it in any of the components.

**Comment by xsaurabhsuthar** [comment]
I have made the necessary changes. Is it okay now?

File: docs.css (hunk 2 above)
Comment (ScoobyDubeyDo) [reply docs.css]: ok

**Review by ScoobyDubeyDo — APPROVED** [review]

**Comment by xsaurabhsuthar** [comment]
Thanks, Aman and Muskaan for the review.

**Pull request merged**